PyPDF2/generic.py

```diff
@@ -1879,6 +1889,16 @@ def bottom(self) -> Optional[FloatObject]:
         """Read-only property accessing the bottom vertical coordinate."""
         return self.get("/Bottom", None)
 
+    @property
+    def color(self) -> Optional[tuple]:
+        """Read-only property accessing the color in (R, G, B) with values 0.0-1.0"""
+        return self.get("/C", (0.0, 0.0, 0.0))
```

Comment:
Could you please adjust a test / write a new test to cover the newly added method?